### PR TITLE
fawe warning

### PR DESCRIPTION
**Pull request by JustDoom**
# Description

Added a warning for fawe

## Checklist

- [x] My contributions follows the style guidelines of this project
- [x] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
- [x] I have run linting on my branch before submitting the PR

### CODE REVIEW

File: tutorials/en_us/PLUGINS/FAWE.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # How to install and use [Fast Async World Edit (FAWE)](https://www.spigotmc.org/resources/fast-async-worldedit.13932/)
 
+## WARNING
+
+FAWE can cause world corruption so use at your own risk.
+
+### How it happens
+
+FAWE asyncs things that shouldnt be async, and then flat out skips other parts of the process to speed it up. FAWE gets is speed from skipping important tasks needed for a stable experience. thats the entire reason it broke on starlight (The lighting engine tuinity uses); it skips the lighting task, and starlight only runs the lighting task once, not thousands like vanilla's. Starlight had to remove a lot of its own optimizations because people on FAWE were complaining about the plugin not working with starlight.
```
Comment (Silverarmor) [inline tutorials/en_us/PLUGINS/FAWE.md:9]: This needs to be split into multiple lines. It has sentences that are too long and do not make sense.

### Read the sentence aloud, does it make sense while you listen to it?

Adiditonally, Tuinity, Vanilla & Starlight should be consistently capitalised.

File: tutorials/en_us/PLUGINS/FAWE.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # How to install and use [Fast Async World Edit (FAWE)](https://www.spigotmc.org/resources/fast-async-worldedit.13932/)
 
+## WARNING
+
+FAWE can cause world corruption so use at your own risk.
```
Comment (Silverarmor) [inline tutorials/en_us/PLUGINS/FAWE.md:5]: There have been reports of FAWE causing world corruption - Use at your own risk

**Review by Silverarmor — CHANGES_REQUESTED** [review]

File: tutorials/en_us/PLUGINS/FAWE.md (hunk 1 above)
Comment (JustDoom) [reply tutorials/en_us/PLUGINS/FAWE.md:9]: should make sense now

Comment (JustDoom) [reply tutorials/en_us/PLUGINS/FAWE.md:9]: ill keep this open just incase it doent

File: tutorials/en_us/PLUGINS/FAWE.md [hunk 3]
```diff
@@ -2,15 +2,17 @@
 
 ## WARNING
 
-FAWE can cause world corruption so use at your own risk.
+FAWE can cause world corruption - Use at your own risk.
 
 ### How it happens
 
-FAWE asyncs things that shouldnt be async, and then flat out skips other parts of the process to speed it up. FAWE gets is speed from skipping important tasks needed for a stable experience. thats the entire reason it broke on starlight (The lighting engine tuinity uses); it skips the lighting task, and starlight only runs the lighting task once, not thousands like vanilla's. Starlight had to remove a lot of its own optimizations because people on FAWE were complaining about the plugin not working with starlight.
+FAWE asyncs tasks that shouldn't be async, then it skips other parts of the process to speed it up. FAWE gets it's speed from skipping important tasks needed for a stable experience. Because FAWE skipped important tasks it broke on Startlight (Tuinity's lighting engine).
```
Comment (Silverarmor) [inline tutorials/en_us/PLUGINS/FAWE.md:9]: Startlight?

**Review by Silverarmor — APPROVED** [review]

**Pull request merged**